src/modules/auth/components/CustomDataContainer.js

```diff
@@ -11,6 +11,18 @@ import {
 	FormGroup
 } from 'reactstrap';
 
+/*
```

Comment:
@petrKavulok Shouldnt this be in some `doc` file?

Reply:
wasn't sure where to put it, since we have docs in `asab-webui/doc` but this is a part of `seacat-admin-webui`. I wasn't sure it was worth creating a `/doc` folder for a documentation of this magnitude. If it should be in the `doc` folder, should the folder be a part of the `/components` folder? or have a different spot in the treemenu?

Reply:
@petrKavulok Nope, it should be in `doc` folder on top level (there already is a `doc` folder)

```
- asab-webui
- conf
- doc <<<< here is the doc folder where you want to add any broader description regarding of any component
- public
- src
- ....
```

btw. docs for asab-webui surely are in `asab-webui/doc`, but this is not the case, innit?

Reply:
you're right, I missed that, I'm sorry :-)